### ISSUE TITLE
Cannot launch Nethackcocoa after duplicating application folder (and then deleting); pid error

### ISSUE THREAD

**Issue by dvmtucson** [bryceco/nethack-cocoa#31]
So I screwed something up when I duplicated the Nethackcocoa folder in my Applications folder (on a MacBook Pro running OX11.6.7).  I have run Nethackcocoa without issue for a couple of weeks prior to this.  I launched from the duplicated folder, quite, then deleted the duplicated folder, but now the original will not launch.

When I launch, it prompts "There are files from a game in progress under your name. Recover?"

I click "No" (because otherwise the program just shuts down).  It lets me choose a player, gives me the "It is written in the book..." window, but then when I close that it asks, "Do you want your possessions identified?" etc.  In the scrolling window it says, "Level #0 pid (58709) doesn't match ours (#####)!" where the ##### is a different number each time I try, and I can see is the pid of the Nethack process in Activity Monitor.  It says I was "You were tricked in the Dungeons of Doom..."

I tried deleting the existing folder, redownloading Nethackcocoa from the App store, but same issue persists.  Help!

**Comment by neal-howland** [comment]
I have had this happen even without duplicating the applications folder. I was able to resolve by deleting the Library/Containers/nethackcocoa folder.
________________________________
From: dvmtucson ***@***.***>
Sent: Wednesday, November 16, 2022 7:00:45 AM
To: bryceco/nethack-cocoa ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [bryceco/nethack-cocoa] Cannot launch Nethackcocoa after duplicating application folder (and then deleting); pid error (Issue #31)


So I screwed something up when I duplicated the Nethackcocoa folder in my Applications folder (on a MacBook Pro running OX11.6.7). I have run Nethackcocoa without issue for a couple of weeks prior to this. I launched from the duplicated folder, quite, then deleted the duplicated folder, but now the original will not launch.

When I launch, it prompts "There are files from a game in progress under your name. Recover?"

I click "No" (because otherwise the program just shuts down). It lets me choose a player, gives me the "It is written in the book..." window, but then when I close that it asks, "Do you want your possessions identified?" etc. In the scrolling window it says, "Level #0 pid (58709) doesn't match ours (#####)!" where the ##### is a different number each time I try, and I can see is the pid of the Nethack process in Activity Monitor. It says I was "You were tricked in the Dungeons of Doom..."

I tried deleting the existing folder, redownloading Nethackcocoa from the App store, but same issue persists. Help!

**Comment by dvmtucson** [comment]
Yes, was able to find the [user]/Library/Containers/Nethack/Data/nethackdir folder finally.  Deleting files in there fixed the problem.  (Trouble is that Library folder is hidden, so had to search "nethackdir" to get in there.)
